test/verk_test.exs

```diff
@@ -129,15 +130,15 @@ defmodule VerkTest do
   end
 
   test "schedule a job with a jid, a queue and a perform_in passing a redis connection" do
-    now = Timex.DateTime.now
-    perform_at = Timex.DateTime.shift(now, days: 1)
+    now = Time.now
+    perform_at = Time.shift(now, 100, :seconds)
```

Comment:
Primarily I changed it because I was trying to limit the scope of what `Verk.Time` needed to implement. The implementation in this PR supports the values that `Datetime.to_unix` uses, of which `:seconds` is the largest granularity (`:days` is supported by Timex but not the standard library functions. Based on the test and implementation `:days` seemed to be chosen arbitrarily as a distance in the future, and since all times are converted to unix time in seconds for comparison and serialization, it seemed like a solid change.